rest/routes/dashboardTemplate.go

```diff
@@ -271,10 +272,32 @@ func DecodeDashboardTemplate(w http.ResponseWriter, r *http.Request) {
 	handleDashboardResponse[models.DashboardTemplate](resp, err, w)
 }
 
+func FilterWidgetMapping(widgetMapping models.WidgetModuleFederationMapping) models.WidgetModuleFederationMapping {
+	filteredWidgets := make(map[models.AvailableWidgets]models.ModuleFederationMetadata)
+
+	for key, value := range widgetMapping {
+		if !featureflags.IsEnabled(value.FeatureFlag) {
```

Comment:
Care to add a comment here, please? Because this is inverted it might be a headscratcher for us when we look at the code in a couple fo weeks. I had to read it twice already 😆